pkg/image/qemu.go

```diff
@@ -133,16 +134,19 @@ func getCacheMode(path string, cacheMode string) (string, error) {
 	}
 
 	var supportDirectIO bool
-	isDevice, err := util.IsDevice(path)
-	if err != nil {
+	var stat unix.Stat_t
+	var err error
+
+	if err = unix.Stat(path, &stat); err != nil {
 		return "", err
 	}
 
-	if isDevice {
+	if (stat.Mode & unix.S_IFMT) == unix.S_IFBLK {
```

Comment:
I am fine with this change but how come you only want it for this single occurrence? did you mean to include this in this commit?